k8sutils/statefulset.go

```diff
@@ -511,7 +511,45 @@ func getVolumeMount(name string, persistenceEnabled *bool, externalConfig *strin
 }
 
 // getProbeInfo generate probe for Redis StatefulSet
-func getProbeInfo(probe *redisv1beta1.Probe) *corev1.Probe {
+func getProbeInfo(params containerParameters, probeType string) *corev1.Probe {
+	probePort := redisPort
+	if params.Role == "sentinel" {
+		probePort = sentinelPort
+	}
+
+	ProbeCommand := []string{
```

Comment:
inside of a function a variable should start with small later in golang (it is not accesseble from outside)

Reply:
Renamed